params.go

```diff
@@ -85,3 +87,24 @@ func ParseBrowsersParam(r *http.Request) (browsers []string, err error) {
 	sort.Strings(browsers)
 	return browsers, nil
 }
+
+// ParsePathsParam returns a set list of test paths to include, or nil if no filter is provided (and all tests should be
+// included). It parses the 'paths' parameter, split on commas, and also checks for the (repeatable) 'path' params.
+func ParsePathsParam(r *http.Request) (paths mapset.Set) {
+	pathParams := r.URL.Query()["path"]
+	pathsParam := r.URL.Query().Get("paths")
+	if len(pathParams) == 0 && pathsParam == "" {
+		return nil
+	}
+
+	paths = mapset.NewSet()
+	for _, path := range pathParams {
+		paths.Add(path)
+	}
+	if browsersParam := pathsParam; browsersParam != "" {
```

Comment:
browsersParam here looks curious, should it just turn into `if pathsParam != "" {`? If so then no guard at all should be necessary, just let the empty string be split.

Reply:
That's the result of copy + paste + intellij refactor out into a variable. Fixed.